model/models/mistral3/model_text.go

```diff
@@ -148,18 +147,6 @@ func NewTextModel(c fs.Config) (*TextModel, error) {
 	}
 
 	textModel := &TextModel{
-		BytePairEncoding: model.NewBytePairEncoding(
```

Comment:
This should be defined in the top-level model rather than specifically in the text model